addon/scope.js

```diff
@@ -0,0 +1,60 @@
+class Scope {
```

Comment:
We could catch all test helpers that Ember defines and raise an exception.

Once they start defining their own, all bets are probably off.